roles/github/defaults/main.yml

```
@@ -23,8 +23,9 @@ github_final_hook: ""
 
 github_buildx_inline_config: ""
 
-github_timeout: 360
+github_buildx_enable_provenance: false
 
+github_timeout: 360
```

```suggestion
github_timeout: 360

```

Looks like my suggestion missed off the newline.